tmtools/io.py

```diff
@@ -7,13 +7,14 @@
 from Bio.PDB.MMCIFParser import MMCIFParser
 
 try:
-    from Bio.PDB.Polypeptide import protein_letters_3to1
+    from Bio.Data.IUPACData import protein_letters_3to1_extended as protein_letters_3to1
 except ImportError:
     # pre 1.80
     from Bio.PDB import protein_letters_3to1
 
 import numpy as np
 
+protein_letters_3to1['UNK'] = 'X'
```

Comment:
@alkorolyov-selvita Thanks for the fix! This does the job but has the side-effect of modifying the `IUPACData` dictionary, which "belongs to" BioPython (so that if this dictionary is used elsewhere in a user's code, the modification will also be visible). I would propose having an explicit fix: e.g. check for `residue.resname == 'UNK'` on the line below, and then insert `X` or the appropriate 1-letter code in the sequence: https://github.com/jvkersch/tmtools/blob/10dde1db6e61590de672c01fd8e3084d320118f8/tmtools/io.py#L119.